portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/ExportResult.java

```diff
@@ -1,23 +1,32 @@
 package org.dataportabilityproject.spi.transfer.provider;
 
+import com.google.common.base.Preconditions;
 import org.dataportabilityproject.spi.transfer.types.ContinuationData;
 import org.dataportabilityproject.types.transfer.models.DataModel;
 
-/** The result of an item export operation. */
+/**
+ * The result of an item export operation.
```

Comment:
nit: this is after all retries, right? can you document that?

Reply:
Done.